src/components/ScoreBoard/AddPlayer/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Consumer } from '../Context';
+
+const AddPlayerForm = () => {
+
+  const playerInput = React.createRef();
+
+  return (
+    <Consumer>
+      { ({ actions }) => {
+        const handleSubmit = (e) => {
+          e.preventDefault();
+          actions.addPlayer(playerInput.current.value);
+          e.currentTarget.reset();
+        }
```

Comment:
There's a cleaner and more readable way to approach this using the `useContext` hook, but I'm guessing this is part of the exercise so let's leave it as it is. Just like to mention it for future reference: https://reactjs.org/docs/hooks-reference.html#usecontext